flux/clusters/k8s-01/keycloak-db/overlays/DEV/postgrescluster-s3-backups.yaml

```diff
@@ -0,0 +1,40 @@
+# https://access.crunchydata.com/documentation/postgres-operator/latest/tutorial/backups/#using-s3
+# https://pgbackrest.org/configuration.html
+
+- op: add
+  path: /spec/backups/pgbackrest/global
+  value:
+    log-level-file: info # enough to see WAL file names
+    log-level-stderr: info # enough to see WAL file names
+    # use /pgbackrest/$NAMESPACE/$CLUSTER_NAME/repoN
+    repo2-path: /pgbackrest/dev-keycloak-db/db-keycloak/repo2
```

Comment:
Please keep in mind that AWS IAM user with correct policy should be created in advance